### DISCUSSION TITLE
Drag and drop to other apps or desktop (also some castomisation)

### DISCUSSION THREAD

**Discussion by nikedayn** [spacedriveapp/spacedrive#2768] [Feature Request]
There are some basic issues that prevent me from switching to Spacedrive as my default file manager. I like the idea of Spacedrive as a "file manager from the future," and I appreciate that I can create different libraries and have full control over which ones I see. This allows me to maintain a clear file management system.

1. The main issue is the lack of drag-and-drop functionality for files or folders into other apps installed on my system. Moreover, I can't drag and drop files between Spacedrive windows.
2. The secondary issue is the limited options in the right-click menu. Specifically, you can't open a folder in a new tab or window from the right-click menu. (I know you can open a folder in a new tab, but it's too difficult to do without hotkeys.)
3. Customization is another problem for me. I can't change folder icons or the icons of certain file types.
4. I always use file managers in small windows, but buttons like "close window," "minimize," and "maximize" are hidden when using the file manager in a smaller window.

Overall, I like the concept of Spacedrive, and it truly feels like the most futuristic file manager for me. However, these basic issues push me to continue using the default Windows file manager.

**Comment by jlecointe** [comment]
Same for the drag and drop functionality..
